packages/e2e-test-app-fabric/windows/RNTesterApp-Fabric/RNTesterApp-Fabric.cpp

```diff
@@ -0,0 +1,283 @@
+
+//
+//  FUNCTION: WndProc(HWND, UINT, WPARAM, LPARAM)
+//
+//  PURPOSE: Processes messages for the main window.
+//
+LRESULT CALLBACK WndProc(HWND hWnd, UINT message, WPARAM wParam, LPARAM lParam) {
+  auto windowData = WindowData::GetFromWindow(hWnd);
+  if (windowData) {
+    auto result = WindowData::GetFromWindow(hWnd)->TranslateMessage(message, wParam, lParam);
+    if (result)
+      return result;
+  }
+
+  switch (message) {
+    case WM_DESTROY: {
+      delete WindowData::GetFromWindow(hWnd);
+      SetProp(hWnd, WindowDataProperty, 0);
+      PostQuitMessage(0);
+      return 0;
+    }
+    case WM_NCCREATE: {
+      auto cs = reinterpret_cast<CREATESTRUCT *>(lParam);
+      auto windowData = static_cast<WindowData *>(cs->lpCreateParams);
+      WINRT_ASSERT(windowData);
+      SetProp(hWnd, WindowDataProperty, reinterpret_cast<HANDLE>(windowData));
+      break;
+    }
+    case WM_GETOBJECT: {
+      if (lParam == UiaRootObjectId) {
+        auto windowData = WindowData::GetFromWindow(hWnd);
+        if (!windowData->m_windowInited)
+          break;
+
+        auto hwndHost = windowData->m_CompositionHwndHost;
+        winrt::com_ptr<IRawElementProviderSimple> spReps;
+        hwndHost.UiaProvider().as(spReps);
+        LRESULT lResult = UiaReturnRawElementProvider(hWnd, wParam, lParam, spReps.get());
+        return lResult;
+      }
+    }
+  }
+
+  return DefWindowProc(hWnd, message, wParam, lParam);
+}
+
+int RunRNTester(int showCmd) {
+  constexpr PCWSTR appName = L"React Native Tester (Composition)";
+
+  auto windowData = std::make_unique<WindowData>(winrt::Microsoft::ReactNative::CompositionHwndHost());
+  HWND hwnd = CreateWindow(
+      c_windowClassName,
+      appName,
+      WS_OVERLAPPEDWINDOW,
+      CW_USEDEFAULT,
+      CW_USEDEFAULT,
+      CW_USEDEFAULT,
+      CW_USEDEFAULT,
+      nullptr,
+      nullptr,
+      WindowData::s_instance,
+      windowData.get());
+
+  WINRT_VERIFY(hwnd);
+  winrt::check_win32(!hwnd);
+
+  windowData.release();
+
+  ShowWindow(hwnd, showCmd);
+  UpdateWindow(hwnd);
+  SetFocus(hwnd);
+  WindowData::GetFromWindow(hwnd)->RenderApp(hwnd);
+
+  HACCEL hAccelTable = LoadAccelerators(WindowData::s_instance, MAKEINTRESOURCE(IDC_RNTESTER_COMPOSITION));
+
+  MSG msg = {};
+  while (GetMessage(&msg, nullptr, 0, 0)) {
+    if (!TranslateAccelerator(hwnd, hAccelTable, &msg)) {
+      TranslateMessage(&msg);
+      DispatchMessage(&msg);
+    }
+  }
+  return static_cast<int>(msg.wParam);
+}
+
+_Use_decl_annotations_ int CALLBACK WinMain(HINSTANCE instance, HINSTANCE, PSTR /* commandLine */, int showCmd) {
+  WNDCLASSEXW wcex = {};
+  wcex.cbSize = sizeof(WNDCLASSEX);
+  wcex.style = CS_HREDRAW | CS_VREDRAW;
+  wcex.lpfnWndProc = &WndProc;
+  wcex.cbClsExtra = DLGWINDOWEXTRA;
+  wcex.cbWndExtra = sizeof(WindowData *);
+  wcex.hInstance = WindowData::s_instance;
+  wcex.hCursor = LoadCursor(nullptr, IDC_ARROW);
+  wcex.hbrBackground = (HBRUSH)(COLOR_WINDOW + 1);
```

Comment:
```suggestion
  wcex.hbrBackground = static_cast<HBRUSH>(COLOR_WINDOW + 1);
```

Reply:
This code failed with the error that it was not possible to static cast between these two types.